Rebuild and bump

pkgw opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

pkgw committed:
conda-forge.yml: Python 3.7 update

pkgw committed:
recipe/meta.yaml: bump to 1.1.6

pkgw committed:
MNT: Re-rendered with conda-build 3.18.2, conda-smithy 3.3.4, and conda-forge-pinning 2019.05.26

pkgw committed:
recipe/meta.yaml: tests require git

pkgw committed:
recipe/meta.yaml: update magic Python install line

pkgw committed:
recipe/meta.yaml: these deps should be host, not build

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

pkgw committed:
recipe/meta.yaml: attempt to fix Azure Linux builds

pkgw commented:
I think that to get the Windows builds to work, we need conda-forge/schwimmbad-feedstock#3.

pkgw committed:
recipe/meta.yaml: looks like we also need SSH for Azure Linux to build

pkgw committed:
recipe/meta.yaml: silly commit to trigger rebuild

I think we should pass CI now, but I can't figure out how to tell Azure to
retry the failed Windows/Python3.7 build. So, here we turn a couple of HTTP
URLs into HTTPS.

pkgw committed:
recipe/meta.yaml: sigh, no mpi4py on Windows here, either

Since it's not available for Python 3.7.

pkgw commented:
OK, the Azure failure is a spurious thing that has succeeded elsewhere, so we are OK to go here!
